charts/karmada/templates/_karmada_bootstrap_token_configuration.tpl

```diff
@@ -0,0 +1,202 @@
+{{- define "karmada.bootstrapToken.configuration" -}}
+{{- $name := include "karmada.name" . -}}
+{{- $namespace := include "karmada.namespace" . -}}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-info
+  namespace: kube-public
+data:
+  kubeconfig: |
+    apiVersion: v1
+    clusters:
+    - cluster:
+        {{- include "karmada.kubeconfig.caData" . | nindent 8 }}
+        server: https://{{ $name }}-apiserver.{{ $namespace }}.svc.{{ .Values.clusterDomain }}:5443
```

Comment:
https://github.com/karmada-io/karmada/blob/master/charts/karmada/values.yaml#L378
https://github.com/karmada-io/karmada/blob/master/charts/karmada/values.yaml#L382

Can we use this param `apiServer.nodePort` in values.yaml to fill the server port.
And add `HostIP` params to values.yaml to set karmada apiserver host.

In this way, user is not need to update the kubeconfig manually. How do you think?

Reply:
It is a good way, but it also is same as `kubeconfig` currently, so it could be designed for more configurable in next PR at same time. This PR focus on configuration for `register command`?

I will file an issue to trace this requirement, what do you think?

Reply:
ok, I'm with you.